docs/guides/UpgardeGuide-v5/UpgradePart2.md

```diff
@@ -0,0 +1,1012 @@
+        ✓ is accessible
+        ✓ [default] is within the mismatch tolerance // 1 screenshot
+
+      > [tiny] Example - table - table with first item selected
+        ✓ clicks row 1
+        ✓ [default] is accessible & is within the mismatch tolerance // 1 screenshot
+
+      > [tiny] Example - table - table with second item selected
+        ✓ clicks row 2
+        ✓ is accessible
+        ✓ [default] is within the mismatch tolerance // 1 screenshot
+
+> [huge] Example
+  ~ [huge] Example - before() // 1 viewport resize
+
+    > [huge] Example - input
+      ~ [huge] Example - input - before()
+        ✓ [default] is accessible & is within the mismatch tolerance // 1 screenshot
+        ✓ enters text
+        ✓ [with input] is accessible & is within the mismatch tolerance // 1 screenshot
+
+    > [huge] Example - popup
+      ~ [huge]Example - popup - before()
+        ✓ [default] is accessible & is within the mismatch tolerance // 1 screenshot
+        ✓ click popup content // 1 screenshot
+
+    > [huge] Example - table
+      ~ [huge] Example - table - before()
+
+      > [huge] Example - table - table default
+        ✓ is accessible
+        ✓ [default] is within the mismatch tolerance // 1 screenshot
+
+      > [huge] Example - table - table with first item selected
+        ✓ clicks row 1
+        ✓ [default] is accessible & is within the mismatch tolerance // 1 screenshot
+
+      > [huge] Example - table - table with second item selected
+        ✓ clicks row 2
+        ✓ is accessible
+        ✓ [default] is within the mismatch tolerance // 1 screenshot
+
+24 passing (50ms)
+```
+
+</details>
+
+<details>
+<summary>Generated Screenshots</summary>
+
+```diff
+/reference
+  /en
+    /chrome_tiny
+      /check-spec
++       /input[default].png     <-- added
++       /input[with_input].png  <-- added
+        /table_default[default].png
++       /table_with_first_item_selected[row_1_selected].png <-- added
+        /table_with_second_item_selected[row_2_selected].png
+        /popup[default].png
+-       /click_popup_content[default].png  <-- removed
++       /popup[click_popup_content].png    <-- added
+    /chrome_huge
+      /check-spec
++       /input[default].png     <-- updated to be terra huge size
++       /input[with_input].png  <-- updated to be terra huge size
+        /table_default[default].png
+        /table_with_first_item_selected[row_1_selected].png
+        /table_with_second_item_selected[row_2_selected].png
+        /popup[default].png
+-       /click_popup_content[default].png  <-- removed
++       /popup[click_popup_content].png    <-- added
+    /firefox_tiny  <-- same as chrome_tiny
+    /firefox_huge  <-- same as chrome_huge
+```
+</details>
+
+## Test Update Gotchas
+So these test updates are all hunky dory, however there are a few things that might confuse you in your screenshots. 
+
+### 1. Orange Focus Glow?
+
+This is the browser focus glow color on native html element defined by Google Chrome.
```

Comment:
should we mention something about chrome versions?

Reply:
This was mentioned in the Changes doc.